src/Symfony/Component/Translation/Bridge/Lokalise/Tests/LokaliseProviderTest.php

```diff
@@ -562,6 +562,7 @@ public function testReadForOneLocaleAndOneDomain(string $locale, string $domain,
                 'filter_langs' => [$locale],
                 'filter_filenames' => [$domain.'.xliff'],
                 'export_empty_as' => 'skip',
+                'replace_breaks' => false,
```

Comment:
What about extending the test with your use case?

Reply:
Since we don't contact Lokalise API during tests, is it really useful?